components/shared/Banner.js

```diff
@@ -0,0 +1,25 @@
+/* @author Charles Liu */
```

Comment:
Nice Work Migrating all the reusable components to a shared/ directory

Reply:
thanks bro